service/middleware.go

```diff
@@ -448,6 +448,26 @@ func (l *Logger) ServeHTTP(rw http.ResponseWriter, r *http.Request, next http.Ha
 		"path":    r.URL.Path,
 	})
 
+	defer func() {
+		if err := recover(); err != nil {
+			if rw.Header().Get("Content-Type") == "" {
+				rw.Header().Set("Content-Type", "text/plain; charset=utf-8")
+			}
+
+			rw.WriteHeader(http.StatusInternalServerError)
+
+			grip.Critical(message.Fields{
+				"stack":    message.NewStack(2, "").Raw(),
+				"panic":    err,
```

Comment:
Is there one splunk query that finds panics across web/runner/agent? We should standardize the message for this.

Reply:
there isn't. but this uses the same fields (i.e. panic/stack) as the messages in grip's recovery package (which is what every other tool uses.) I wanted to *not* use that here so that we could add request context.